torch/jit/__init__.py

```diff
@@ -82,9 +82,10 @@ def annotate(the_type, the_value):
 
     Though TorchScript can infer correct type for most Python expressions, there are some cases where
     type inference can be wrong, including:
-    - Empty containers like `[]` and `{}`, which TorchScript assumes to be container of `Tensor`s
+
+    - Empty containers like `[]` and `{}`, which TorchScript assumes to be container of `Tensor`\ s
```

Comment:
Not sure why you are adding backslash here, isn't it supposed to be a simple plural (and `Tensor`s markup does not really look like a single word , so please either have it wrapped in backticks in its entirety or don't render it as code at all. Perhaps simple italic will do)
cc: @brianjo 
```suggestion
    - Empty containers like `[]` and `{}`, which TorchScript assumes to be container of *Tensors*
```

Reply:
I wanted to write something like `Tensor`s (which is rendered correctly in github when writing `` `Tensor`s ``) which was what the author of that comment surely meant: `Tensor` is a class and we have a few objects of that class.

Reply:
Then again, no big deal, I'll change it for `Tensors` for now, but it'd be good to know how to have something rendered as `Tensor`s if we wanted to do so.

Reply:
So for this, I think the preferred usage is to say tensors generically, without capitalization or code formatting. The same way you might say objects or functions. In this case, I would suggest using something like: 

Empty containers like `[]` and `{}`, which TorchScript assumes to be containers of `Tensor` objects.